python/llm/setup.py

```diff
@@ -370,6 +385,7 @@ def setup_package():
                         "xpu-arl": xpu_lnl_requires,
                         "xpu-arc": xpu_lnl_requires,
                         "xpu-2-6": xpu_26_requires,
+                        "xpu-2-6-arl": xpu_26_arl_requires,
```

Comment:
what's the difference between "xpu-arl" and "xpu-2-6-arl"?

Reply:
`xpu_arl` is previous for PyTorch 2.3.
`xpu_2.6_arl` is for PyTorch 2.6 + ARL-H AOT.